### PR TITLE
1956 Simplified Login Manager initial unit tests

### PR DESCRIPTION
**Pull request by Daniel-Echegaray**

### CODE REVIEW

File: Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/SimplifiedLogin/SimplifiedLoginManagerTests.swift [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ 
+    // MARK: -
+    
+    func testGetSimplifiedLogin_noLoggedInSessionInSharedKeychain (){
+        let mockHTTPClient = MockHTTPClient()
+        let client = Client(configuration: Fixtures.clientConfig, httpClient: mockHTTPClient)
+        let sut = SimplifiedLoginManager(client: client, contextProvider: ASWebAuthSessionContextProvider(), env: .pre, completion: { result in })
+        
+        let expectation = self.expectation(description: "Should fail with SimplifiedLoginError.noLoggedInSessionInSharedKeychain")
+        sut.getSimplifiedLogin(nil, completion: { result in
```
Comment (zamorastp) [inline Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/SimplifiedLogin/SimplifiedLoginManagerTests.swift:15]: should be requestSimplifiedLogin (current name on feature/917-simplified-login branch)

Comment (Daniel-Echegaray) [reply Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/SimplifiedLogin/SimplifiedLoginManagerTests.swift]: I guess I was on the pre-rebased branch. It is updated now 👍

Comment (zamorastp) [reply Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/SimplifiedLogin/SimplifiedLoginManagerTests.swift]: Works now! but why I didn't see it in PR, I don't know 🤔  (there were no merge conflicts to feature branch)

**Review by zamorastp — APPROVED** [review]
Looks good for me

**Pull request merged**